<?xml version="1.0"?>

	
			
		<!--~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~-->
		<!--                                                                       -->
		<!--                                                                       -->		
	    <!--                         HUMAN EXPERT EXPLORATION                      -->
        <!--                       FOR GAS EMISSION MONITORING                     -->
		<!--                                                                       -->
		<!--                                                                       -->
		<!--~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~-->
		<!--                        @AsifArain, Oct 26, 2017                       -->
		<!--~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~-->

<launch>

	<node pkg="plan_execution" 
	      type="human_exploration_node" 
	      name="human_plan_execution" 
	      output="screen">

	    <param name="field_of_view"	        value=  "270.0"/>
	    <param name="sensing_range"	        value=  "15.0"/> <!-- FIXME 8.25 is 10 meter in real -->
	    <param name="num_pan_sweeps"	    value=  "1"/>
	    <param name="num_tilt_sweeps"	    value=  "1"/>
	    <param name="sample_delay"	        value=  "0.1"/>	
	    <param name="experiment_title"	    value=  "prismaforum5-04"/>	
	    <param name="exploration_strategy"	value=  "human-exploration"/>
	
	
	    <param name="map_file"	            value="prismaforum5_map_conf.dat"/>
        <param name="map_size_file"	        value="prismaforum5_mapsize_conf.dat"/>
        <param name="cell_size_file"	    value="prismaforum5_cellsize_conf.dat"/>
        <param name="robot_origin_file"	    value="prismaforum5_origin_conf.dat"/>
	
	    <param name="hard_offset_x"	        value="0.0"/> <!--"0.0"-->
        <param name="hard_offset_y"	        value="0.0"/> <!--"-0.25"-->
        
	

    </node>
</launch>



<!-- NOTES:

rosservice call /ptu_control/sweep "{min_pan: -45.0, max_pan: 45.0, min_tilt: -10.0, max_tilt: -10.0, n_pan: 1, n_tilt: 1, samp_delay: 0.1}"

For sensing range 10m: tilt angle= -5.1655
For sensing range 15m: tilt angle= -3.45
For sensing range 30m: tilt angle= -1.73




-->
